client/jetpack-connect/authorize.js

```diff
@@ -552,9 +559,9 @@ export class JetpackAuthorize extends Component {
 	renderFooterLinks() {
 		const { translate } = this.props;
 		const { authorizeSuccess, isAuthorizing } = this.props.authorizationData;
-		const { blogname, redirectAfterAuth } = this.props.authQuery;
+		const { blogname } = this.props.authQuery;
 		const backToWpAdminLink = (
-			<LoggedOutFormLinkItem href={ redirectAfterAuth }>
+			<LoggedOutFormLinkItem onClick={ this.handleBackToSite }>
```

Comment:
Let's keep the link in the `href`, and make the `onClick` only take care of tracking. This will work since `LoggedOutFormLinkItem` [is just a glorified `<a />`](https://github.com/Automattic/wp-calypso/blob/7a77d9cceb20f4a3cb31b9ecf141fca13bfa73e8/client/components/logged-out-form/link-item.jsx#L12) (and will also work for a lot of other Calypso components, such as <Button />`.)

It is kind of recommended best practice which allows us to avoid redirection helpers, and keeps things more semantic.

Reply:
sounds good! :bowing_man: